packages/zweidenker_heinzelmen/lib/src/link_laucher.dart

```diff
@@ -14,38 +15,59 @@ class LinkLauncher {
   /// if [openExternally] is `true` this will also be launched externally
   /// Use [tabsOptions] to customize the Chrome tabs on Android
   /// Use [safariViewControllerOptions] to customize the SafariController on iOS
+  /// Use [checkTrackingPermission] if the website should be displayed in a SafariViewController on iOS and Displays Cookies
+  /// This can lead to rejection during Apple Review if no Tracking Permission was granted by the user
+  /// If you set this to true you need to declare the usage of `NSUserTrackingUsageDescription` in Info.plist
   Future<void> openWebPage({
     required Uri url,
     bool openExternally = false,
     CustomTabsOptions? tabsOptions,
     SafariViewControllerOptions? safariViewControllerOptions,
+    bool checkTrackingPermission = false,
```

Comment:
Would it make sense to have this set to `true` per default? I would expect for most pages to be opened needing this.

Reply:
I decided against it as that would force all Apps using the LinkLauncher to add the plist entry and thus be a breaking change. Also other Apps may ask for the permission at different points e.g when initialising an Analytics Library so popping this may be weird for users.